exercises/anagram/anagram_test.rb

```diff
@@ -137,6 +104,6 @@ def test_capital_word_is_not_own_anagram
 
   def test_bookkeeping
     skip
-    assert_equal 2, BookKeeping::VERSION
+    assert_equal 3, BookKeeping::VERSION
```

Comment:
Which tests changed?

The commit that does the version update should just include the bits of the file that caused that version upgrade.

These are being hidden by the rest of the noise in the commit due to formatting changes.